Gradata/tests/test_forget_command.py

```
@@ -0,0 +1,123 @@
+    # Take the first word of the first lesson's description as the fuzzy key.
+    # This is robust to whatever the auto-graduation pipeline produced.
+    descs = [(l.description or "") for l in active]
+    candidate = next((d for d in descs if len(d) > 3), "")
+    assert candidate, f"no usable descriptions found: {descs}"
+    needle = candidate.split()[0]
+    out = _run_forget(brain_dir, needle, yes=True, capsys=capsys)
+    assert "Will forget" in out
+
+
+def test_forget_no_matches_returns_clean_error(seeded_brain, capsys):
+    brain, brain_dir = seeded_brain
```

Ruff (0.15.13)

[warning] Unpacked variable `brain` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)